### PR TITLE
added jahanvi's profile in fastn champions

### PR DESCRIPTION
**Pull request by jahanvir**
I have updated the images in dimension 460x460 in folder images/students-program/champions
I have added the fastn champion social links in /student-programs/champions under name jahanvi-raycha.ftd

### CODE REVIEW

File: student-programs/champions/jahanvi-raycha.ftd [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+src: $fastn-assets.files.images.discord.svg
+hover-src: $fastn-assets.files.images.discord-hover.svg
+
+-- common.social-media: 
+link: https://github.com/jahanvir
+src: $fastn-assets.files.images.github-grey.svg
+hover-src: $fastn-assets.files.images.github-grey-hover.svg
+
+-- common.social-media: 
+link: https://twitter.com/
```
Comment (gsalunke) [inline student-programs/champions/jahanvi-raycha.ftd:71]: @jahanvir pl add your twitter profile id here.

**Review by gsalunke — CHANGES_REQUESTED** [review]
@jahanvir can you also add your name in this list? https://fastncom-git-fork-jahanvir-main-fifthtry.vercel.app/champions/
I can see you here: https://fastncom-git-fork-jahanvir-main-fifthtry.vercel.app/student-programs/champions/jahanvi-raycha/
your profile should be visible also on  https://fastncom-git-fork-jahanvir-main-fifthtry.vercel.app/champions/ page

File: student-programs/champions/jahanvi-raycha.ftd (hunk 1 above)
Comment (jahanvir) [reply student-programs/champions/jahanvi-raycha.ftd:71]: @gsalunke , I am not using twitter

Comment (gsalunke) [reply student-programs/champions/jahanvi-raycha.ftd:71]: > @gsalunke , I am not using twitter

then, please, remove the link for now, in future if you start using it, you can raise a separate PR.

File: student-programs/champions/jahanvi-raycha.ftd [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+src: $fastn-assets.files.images.twitter.svg
+hover-src: $fastn-assets.files.images.twitter-hover.svg
```
Comment (gsalunke) [inline student-programs/champions/jahanvi-raycha.ftd:73]:
```suggestion
```

**Review by gsalunke — CHANGES_REQUESTED** [review]

File: student-programs/champions/jahanvi-raycha.ftd [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+link: https://discord.com/invite/jahanvi
+src: $fastn-assets.files.images.discord.svg
+hover-src: $fastn-assets.files.images.discord-hover.svg
+
+-- common.social-media: 
+link: https://github.com/jahanvir
+src: $fastn-assets.files.images.github-grey.svg
+hover-src: $fastn-assets.files.images.github-grey-hover.svg
+
+
```
Comment (gsalunke) [inline student-programs/champions/jahanvi-raycha.ftd:70]:
```suggestion
```

**Review by gsalunke — CHANGES_REQUESTED** [review]

**Review by gsalunke — APPROVED** [review]

**Pull request merged**